src/js/controllers/ctrl-media-import.js

```diff
@@ -195,6 +231,7 @@ app.controller('MediaImportCtrl', function ($scope, $window, $timeout) {
 			case 'mp3': // intentional case fall-through
 			case 'wav': // intentional case fall-through
 			case 'ogg': // intentional case fall-through
+			case 'obj': // intentional case fall-through
```

Comment:
This is determining which thumbnail image to show in the media selector, so `obj` should probably not have the same thumbnail as audio files.

Give `obj` its own case and make a new image to serve as the thumbnail for files of this type.

Reply:
I'll search for a img to add move the case.

Reply:
There is an image provided in the version of this code that's already on production. As stated in my review, it's fine to override this change with whatever was done on master.